extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -286,6 +289,18 @@ function _checkIfCanAddMeasurementsToDisplaySet(
       continue;
     }
 
+    // if it is a SCOORD3D measurement we can also just add the SR annotation
+    if (
```

Comment:
There's already a condition is3DSR below which will be true independently of this code

Reply:
The files that are being generated by dcmsj/cs3d do not get into the first condition.

Reply:
I don't think I'd include is3DSR here - that doesn't actually matter if you have SCOORD3D it can just be handled.